android/src/main/java/io/mosip/tuvali/transfer/TransferReportRequest.kt

```diff
@@ -1,7 +1,7 @@
 package io.mosip.tuvali.transfer
 
-class Semaphore {
-  enum class SemaphoreMarker {
+class TransferReportRequest {
+  enum class TransferReportRequestMarker {
```

Comment:
Need a better name for Enum class. As it's already under Transfer Report Request, Still having transferReport in the name is a repetition